### PR TITLE
Language: remove whitelist from the code

### PR DESCRIPTION
**Pull request by Halamix2**
**Description**

Changes proposed in this pull request:

- remove `whitelist` from clustercollector

**Related issue(s)**

https://github.com/kyma-project/community/issues/515

### CODE REVIEW

**Review by dekiel — APPROVED** [review]

**Review by kazydek — APPROVED** [review]

File: prow/images/prow-tools/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.15.5-alpine3.12 AS builder
+FROM golang:1.16.2-alpine3.13 AS builder
```
Comment (Ressetkk) [inline prow/images/prow-tools/Dockerfile:1]: Does the bump really is necessary? Shouldn't we bump all Go images in separate PR?

**Review by Ressetkk — APPROVED** [review]
Approved with one comment

File: prow/images/prow-tools/Dockerfile (hunk 1 above)
Comment (Halamix2) [reply prow/images/prow-tools/Dockerfile:1]: I needed to bump this one in order to be able to remove whitelist argument form prowjobs definitions that are using program from this image
